composer.json

```diff
@@ -35,7 +35,7 @@
     },
     "require-dev": {
         "roave/security-advisories": "dev-master",
-        "orchestra/testbench": "^8.0|^9.0",
+        "orchestra/testbench": "^8.0|^9.2",
```

Comment:
This change is required to comply with [CVE-2024-52301 ](https://github.com/laravel/framework/security/advisories/GHSA-gv7v-rgg6-548h) enforced by `roave/security-advisories`.